docs/api-specs.md

```diff
@@ -765,6 +765,10 @@ This endpoint is used to get the price of a namespace, while explicitly
 indicating the cryptocurrency units.  This is because going forward, namespaces
 are not necessarily priced in Bitcoin.
 
+The `amount` given will be in the smallest possible units of the currency.  For
```

Comment:
> Specify the `amount` n the smallest possible units of the currency.  For
> `BTC`, the `amount` is in satoshis.  For `STACKS`, the `amount` is in
> micro-Stacks.

Technical documentation, unlike some other forms of writing, speaks to the reader in the present -- the user has a problem to solve and consults the documentation to find out what to do.

Reply:
Today, new namespaces must be registered using the STACKS currency, so the text here for the price endpoints is correct, I think:

e.g., https://core.blockstack.org/v2/prices/namespaces/trytobuyme

returns:

```json
{
  "amount": "640000000", 
  "units": "STACKS"
}
```